spec/client_spec.rb

```diff
@@ -126,19 +126,18 @@
   end
 
   context '#small_business?' do
-    it "should verify that vendor in the system is a small business with 5-digit naics code" do
-      response = client.small_business?(duns: nine_duns, naicsCode: naics_code)
-      expect(response).to be(true)
-    end
-
-    it "should verify that vendor in the system is a small business with full 6-digit naics code" do
-      response = client.small_business?(duns: nine_duns, naicsCode: full_naics_code)
-      expect(response).to be(true)
+    context 'the DUNS belongs to a big business' do
+      it 'should return false' do
+        response = client.small_business?(duns: big_biz_duns)
```

Comment:
I *believe* it's the DUNS of a known large business, but that's a good question whose answer should be documented here.